lib/settings/defaults.js

```diff
@@ -1,19 +1,37 @@
 const filename_format = function ({testSuite = '', testCase = '', isError = false, dateObject = new Date()} = {}) {
   const fileName = [];
-  const dateParts = dateObject.toString().replace(/:/g, '').split(' ');
-  dateParts.shift();
-
-  const dateStamp = dateParts.slice(0, 5).join('-');
   if (testSuite) {
     fileName.push(testSuite);
   }
   if (testCase) {
     fileName.push(testCase);
   }
+  const dateStamp = getDateStamp(dateObject);
 
   return `${fileName.join('/')}${isError ? '_ERROR' : '_FAILED'}_${dateStamp}.png`;
 };
 
+const folder_format = function({failed, errors, dateObject = new Date()} = {}) {
+  const dateStamp = getDateStamp(dateObject);
```

Comment:
I don't think we should have this as prefix, as it will make it harder to sort the report folder names.